src/emailservice/email_server.rb

```diff
@@ -0,0 +1,22 @@
+require "ostruct"
+require "pony"
+require "sinatra"
+
+require "opentelemetry/sdk"
+require "opentelemetry/exporter/otlp"
+require "opentelemetry/instrumentation/sinatra"
+
+OpenTelemetry::SDK.configure do |c|
+  c.use "OpenTelemetry::Instrumentation::Sinatra"
```

Comment:
I think `c.use_all` is the use that we want to steer folks towards.

Reply:
Ah, good point.